Answers.md

```diff
@@ -1,6 +1,30 @@
-1.  Name 3 JavaScript Array/Object Methods that do not produce side-effects? Which method do we use to create a new object while extending the properties of another object?
-1.  Describe `actions`, `reducers` and the `store` and their role in Redux. What does each piece do? Why is the store known as a 'single source of truth' in a redux application?
-1.  What is the difference between Application state and Component state? When would be a good time to use one over the other?
-1.  What is middleware?
-1.  Describe `redux-thunk`, what does it allow us to do? How does it change our `action-creators`?
-1.  Which `react-redux` method links up our `components` with our `redux store`?
+Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions locatied in This document. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
+
+1.  Name 3 JavaScript Array/Object Methods that do not produce side-effects. Which method do we use to create a new object while extending the properties of another object?
+
+    - Side Effects: When a function changes variables outside of the scope of that specific function.
+    - `.map()`, `.filter()`, and `Object.create()` are three examples of Array/Object methods that don't produce side-effects. Each of these methods produce a new Array or Object without changing the original.
+    - To create a new object while extending the properties of another, we can use `Object.create()`.
```

Comment:
Hmmm I think you're right about `Object.create()`.. I've been saying it's `Object.assign()` for a while now but I'm still confused as to the difference between them. Check it out:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create